src/HospitalLibrary/Managers/Dtos/DoctorsPopularityDTO.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalLibrary.Managers.Dtos
+{
+    public class DoctorsPopularityDTO
+    {
+        public int DoctorID { set; get; }
+
+        public string DoctorName { set; get; }
+
+        public string DoctorSurname { set; get; }
+
+        public int NumberOfPatientsPicked { set; get; }
+
+        DoctorsPopularityDTO()
+        {
+
+        }
+        DoctorsPopularityDTO(int doctorID, int numberOfPatientsPicked)
+        {
+            DoctorID = doctorID;
+            NumberOfPatientsPicked = numberOfPatientsPicked;
+        }
```

Comment:
add name and surname to constructor or delete name and surname and rename name and surname(Doctor part is redundant)